Datil-prod Changes + Some Fixes (#318)
LIT-Protocol/docs: docs/concepts/capacity-credits-concept.md

A `Capacity Credits NFT` can be very easily minted from the Lit Explorer. So, here we will show how you can mint it using `contracts-sdk`. You can download the `contracts-sdk` from `npm` [here](https://www.npmjs.com/package/@lit-protocol/contracts-sdk).

> You’ll also need some 'tstLPX' tokens for minting. These are test tokens that hold no real value and should only be used to pay for usage on Datil-prod. `tstLPX` should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).

Reviewer: maybe we should just update the URL to direct to the new one though

> You’ll also need some 'tstLPX' tokens for minting. These are test tokens that hold no real value and should only be used to pay for usage on Datil-prod. `tstLPX` should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).

Reviewer: well, since we've told some select folks to continue using Habanero, maybe unneccessary right now. The call out box is enough i think. But we should ensure this link directs to the Yellowstone faucet

> You’ll also need some 'tstLPX' tokens for minting. These are test tokens that hold no real value and should only be used to pay for usage on Datil-prod. `tstLPX` should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).

Author: Updated all of the faucet links to Yellowstone

You can read more about Session Signatures [here](../sdk/authentication/session-sigs/intro.md).

> >**Note**: With the migration to SDK v3, the use of `AuthSigs` is no longer recommended for various tasks such as access control, encryption, minting PKPs, Lit Action signing with the PKP, etc. This change is due to the requirement for `capacityDelegationAuthSig` in signatures, as capacity credits are now required on both `Datil` and `Datil-test` networks. We strongly advise users to transition to using `SessionSigs` alongside `capacityDelegationAuthSig`, as `AuthSigs` lack the necessary capabilities to delegate capacity credits moving forward.  You may continue to use `AuthSigs` for the time being, but the limitation is that the `AuthSig` must have capacity credits to use the service.  This means every one of your users must buy capacity credits from Lit.  Instead, you can buy capacity credits centrally and delegate their use to your users. You can generate a `SessionSig` with the help of `capacityDelegationAuthSig` object in the following way (and replace it with `AuthSig` in your project):

Reviewer 2: ```suggestion
```

```javascript
import { LitNetwork } from "@lit-protocol/constants";

  const litNodeClient = new LitNodeClient({

>       litNetwork: LitNetwork.DatilTest,

Reviewer: did we change the lit network initialization convention?

Author: I think we want to start doing this, this would mean if a network name needs to be changed for some reason, user's code will update alongside the SDK

Reviewer: cool yea, that sounds good to me. I more just wanted to verify that if a dev passed it into the SDK it wouldn't throw an error :)

### **Best Practices**

- **Capacity Management**: Keep an eye on your usage limit and expiration date.

> - **Understanding Limits**: Be aware that there are no free requests on `Datil` mainnet. On `Datil-test` testnet, free tier rate limit is set to three free requests per day. You can use Capacity credits to reserve more usage on these networks.

Reviewer 2: ```suggestion
- **Understanding Limits**: Be aware that there are no free requests on `Datil` mainnet or the `Datil-test` testnet. You must use Capacity credits to reserve usage on these networks.
```
